### PR TITLE
Issue#2 register screen mockup

### LINKED ISSUE #2: Register screen mockup
# REQUIREMENTS
- Implement the register screen mockup (without functionality)
- Register mockup must respect the attached register's design structure

# DESIGN
Provided design

# Criteria
- [x] First name field
- [x] Email filed (validated as email)
- [x] Password field
- [x] Check boxes to terms and newsletter
- [x] Sign up button
- [x] Sign up button with google
- [x] Login reference

### PR DESCRIPTION
**Pull request by Namoku**
# Description
What did you implemented/Why did you implemented this:
 - Add form design
 - Add buttons
 - Add overlay
 - Add checkboxes

  ## Screenshots 
image

### CODE REVIEW

**Review by danthek — DISMISSED** [review]

File: ios/kata05/Info.plist [hunk 1]
```diff
@@ -51,5 +51,24 @@
...
+		<string>FontAwesome5_Regular.ttf</string>
+		<string>FontAwesome5_Solid.ttf</string>
+		<string>Fontisto.ttf</string>
+		<string>Foundation.ttf</string>
+		<string>Ionicons.ttf</string>
+		<string>MaterialCommunityIcons.ttf</string>
+		<string>MaterialIcons.ttf</string>
+		<string>Octicons.ttf</string>
+		<string>SimpleLineIcons.ttf</string>
+		<string>Zocial.ttf</string>
```
Comment (YilKanda) [inline ios/kata05/Info.plist]: Why are you adding multiple fonts?

**Review by YilKanda — CHANGES_REQUESTED** [review]
My observation in this PR is that you are installing too many typefaces. It is advisable to have a guideline of typefaces to use. Example, I leave you the link of [FontJoy](https://fontjoy.com/), where you can make a combination of 3 fonts. But remember that the combination of many typographic fonts only incites confusion and is not intuitive for Internet users, in addition the use of a certain font is also essential since each one has a personality and transmits a message.

I imagine all these fonts are added by icons, but do you really take up all the icons from those fonts?

File: ios/kata05/Info.plist (hunk 1 above)
Comment (Darkboy5120) [reply ios/kata05/Info.plist]: We miss that detail, we have just fixed!, we are focusing in android version right now, but we will use the ionicons ttf, in both versions in order to use only one, as you said, we dont want to confuse the user, thanks for the feedback

**Review by Darkboy5120 — APPROVED** [review]
Thanks for the feedback!, we have just fixed that, we will take fonts in consideration from now, thanks again :)

**Review by Darkboy5120 — DISMISSED** [review]
Thanks to you Yill for your feedback, we will take in consideration from now using only one or very few fonts

File: src/screens/LoginScreen.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+  },
+  alternateSignSeparator: {
+    paddingBottom: 10,
+    textAlign: 'center',
+  },
+  checkboxContainer: {
+    paddingVertical: 20,
+  },
+});
+
```
Comment (YilKanda) [inline src/screens/LoginScreen.js]: Can you call these styles in a separate file?

File: src/components/atoms/ToLoginMessage.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import {View, Text, Pressable} from 'react-native';
+import style from '../../styles/ToLoginMessage';
+
+export const ToLoginMessage = () => {
+  const goToLoginScreen = () => {
+    console.log('to login screen');
```
Comment (YilKanda) [inline src/components/atoms/ToLoginMessage.js]: Do not upload console.log() code to GitHub, if you want to display a message I recommend using the [Alert](https://reactnative.dev/docs/alert) component.

**Review by YilKanda — CHANGES_REQUESTED** [review]

File: src/theme/light.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
+import {StyleSheet} from 'react-native';
+
+/*const Colors = {
+  primaryColor: '#5566dd',
+  disabledColor: '#cccccc',
+};*/
```
Comment (YilKanda) [inline src/theme/light.js]: Try not to leave commented code on GitHub. If you are not using it, it is better to delete it.

Comment (Namoku) [reply src/theme/light.js]: Done, I've already deleted.

**Review by Namoku — COMMENTED** [review]
resolved

**Review by Namoku — COMMENTED** [review]
done

**Review by YilKanda — APPROVED** [review]

**Review by DemisRincon — APPROVED** [review]
It's ok, but the imports of the styles could be indexed (exported all from an index) as your molecules component, in general, the code is o but keep your code structure consistent index every export or remove the indexed structure

**Pull request merged**